src/ControllerProvider/controller.json

```diff
@@ -1,10 +1,10 @@
 {
-  "ip": "192.168.1.106",
+  "ip": "34.83.89.255",
   "port": "51121",
   "refresh": 3000,
   "dev": true,
   "user": {
-    "email": "user@domain.com",
-    "password": "#Bugs4Fun"
+    "email": "orange@edgeworx.io",
```

Comment:
Don't commit files on a public Github with private credentials.
The current ones are default/placeholders.